src/set_theory/ordinal/arithmetic.lean

```diff
@@ -1112,6 +1110,17 @@ let hs' := bdd_above_iff_small.2 hs in
   ((cSup_le_iff' hs').2 (le_sup_shrink_equiv hs)).antisymm'
   (sup_le (λ x, le_cSup hs' (subtype.mem _)))
 
+theorem Sup_ord {s : set cardinal.{u}} (hs : bdd_above s) : Sup (ord '' s) = (Sup s).ord :=
```

Comment:
Maybe
```suggestion
theorem Sup_ord_image {s : set cardinal.{u}} (hs : bdd_above s) : Sup (ord '' s) = (Sup s).ord :=
```

Reply:
Another solution might be to revert the arguments, so as to match `Sup_lift` and `Inf_lift`. There's an argument to be made that the LHS is "simpler" than the RHS, since it can allow further simplification if `s` is defined via `set_of`.

Reply:
I've decided to revert the equality.